src/Maestro/Maestro.Web/Api/v2019_01_16/Models/BuildData.cs

```diff
@@ -0,0 +1,62 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using Maestro.Web.Api.v2018_07_16.Models;
+using Microsoft.AspNetCore.ApiVersioning;
+
+namespace Maestro.Web.Api.v2019_01_16.Models
+{
+    public class BuildData : Maestro.Web.Api.v2018_07_16.Models.BuildData
```

Comment:
This one should probably also not inherit the previous because you need to remove properties.

Reply:
I'll change it.